common/src/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { Section as InternalSection, ExternalSectionProps } from "./Section";
+import "../../css/Accordion.css";
+
+export interface AccordionProps {
+  defaultOpenIndex?: number;
+}
```

Comment:
The design that Josh submitted has a simple accordion in addition to the complex accordion that you have implemented.  I wonder if the simple accordion needs to be implemented as well with another prop?